server/models/Notification.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const notificationSchema = new Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+
+  notifications: [
```

Comment:
Please explain why notifications is an array? Since we have the 'sender' (which signifies multiple users notifications stored within the same array) field within this array I am not sure how you plan to implement this design? 🤔

Reply:
I am thinking, there would be three at least types of notifications (serviceRequest, serviceAvailable, message) so a user could get different types of notifications.